src/js/utils.js

```diff
@@ -10,3 +15,52 @@ export default function debounce(fn, delay) {
     }, delay);
   };
 }
+
+export function sendMessage() {
+  $('.js-sendMsg').each((_, btn) => {
```

Comment:
IMHO
```js
$('.js-sendMsg').each((_, btn) => {
    $(btn).click(() => {
      const form = $(btn).parents('form')[0];

// is equal to

  $('.js-sendMsg').click(() => {
    const form = $(this).parent('form');
```

- [ ] simplify click
- [ ] parent vs parents

Reply:
Hmm, three things:
1) `$(this)` with arrow functions doesn't work in this sense (I think it's because an arrow function doesn't posses its own `this`) and Eslint "prefers" arrow functions.
2)  `parent` doesn't work exactly the same- it stops at level that is marked by selector in parentheses, but doesn't include this level (it includes up to `level - 1`). So it would require to set as a selector container `div`, that is "higher in the ladder" than the form, which then would require additional code, to get the `form` we're interested in, and in general could result in more complications with possible future changes.
3) We have 2 submit buttons for `contact-form`, `.each()` ensures all of them have added the event listeners.
I've tried similar approach to the one you recommend before, and now once more, but it seems it just doesn't work the way it's supposed to...